mixer/pkg/runtime/config/config_test.go

```diff
@@ -529,6 +529,86 @@ Attributes:
 `,
 	},
 
+	{
+		Name: "basic handler cr config",
+		Events1: []*store.Event{
+			{
+				Key: store.Key{
+					Name:      "a1",
+					Namespace: "ns",
+					Kind:      "handler",
+				},
+				Type: store.Update,
+				Value: &store.Resource{
+					Spec: &configpb.Handler{
+						Name:            "a1",
+						CompiledAdapter: "adapter1",
+						Adapter:         "adapter1",
```

Comment:
Should this be a validation error? Adapter1 is declared as a static adapter and used as an OOP adapter?